### ISSUE TITLE
SaaS UI: AWS EC2 Deployment fails with 'Permissions 0664 are too open'

### ISSUE THREAD

**Issue by duelle** [xlab-si/xopera-opera#184]
I try to deploy an EC2 instance via the Opera SaaS WebUI.

I have three secrets set up in my workspace:
- AWS Credentials `/root/.aws/credentials`
- EC2 SSH Private Key `/root/.aws/ec2_ssh`
- SSH Config `/root/.ssh/config`
```
Host *.amazonaws.com
  User ubuntu
  IdentityFile /root/.aws/ec2_ssh
  StrictHostKeyChecking no
  UserKnownHostsFile /dev/null
```

The creation of the project and the instantiation of the EC2 instance seems to work fine, but once Ansible tries to connect to the newly deployed EC2 instance, it fails with the following message:
```
"msg": "timed out waiting for ping module test: Failed to connect to the host via ssh: Warning: Permanently added 'ec2-****.eu-west-1.compute.amazonaws.com,****' (ECDSA) to the list of known hosts.
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@         WARNING: UNPROTECTED PRIVATE KEY FILE!          @
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
Permissions 0664 for '/root/.aws/ec2_ssh' are too open.
It is required that your private key files are NOT accessible by others.
This private key will be ignored.
Load key \"/root/.aws/ec2_ssh\": bad permissions\r\nubuntu@ec2-****.eu-west-1.compute.amazonaws.com: Permission denied (publickey)."
```
@sstanovnik

**Comment by sstanovnik** [comment]
Secret file permission management has been implemented, but has not yet been deployed to saas-xopera.xlab.si. We're holding back to release (and test) a larger feature update that will also support scaling operations. This is expected to be deployed soon™.

**Comment by duelle** [comment]
@sstanovnik Are there any updates on the schedule on the release of Opera v0.6.5 as well as its deployment to the Opera SaaS service?

**Comment by anzoman** [comment]
@duelle the new `0.6.5` opera CLI release will come right after we merge #187 and #188.

**Comment by sstanovnik** [comment]
Version `0.6.5` of `xopera-opera` has been released today, followed by version `0.4.0` of `xopera-api` and a new deployment of `xopera-saas` which uses the new features.

You can now specify file modes through the GUI, as shown below. This is, of course, also possible through the API.

Editing secrets is not yet possible, so you will have to create new ones with your preferred file permissions.

![image](https://user-images.githubusercontent.com/8071628/113193102-e49d6200-925f-11eb-8458-ac12158c77fb.png)
